recommendation/project.all.json

```diff
@@ -40,6 +40,12 @@
               "Type": "ProjectFile",
               "Value": "",
               "Description": "Migrate csproj file to core"
+            },
+            {
+              "Name": "WebFormsPorting",
+              "Type": "ProjectType",
+              "Value": "",
+              "Description": "Perform WebForms specific porting steps"
             }
```

Comment:
Is this valid if we're porting to .netstandard2.1? 

.NET Standard apps are compatible with both .NET Framework and .NET Core runtimes, so my instinct here is that Blazor will not be available to apps targeting .NET Standard.

Reply:
Good catch here, after some looking it appears that .NET Standard supports Blazor WebAssembly but not Blazor Server (which we use) so I'll remove this one